cmd/manager/main.go

```diff
@@ -129,8 +140,9 @@ func main() {
 
 	// CreateServiceMonitors will automatically create the prometheus-operator ServiceMonitor resources
 	// necessary to configure Prometheus to scrape metrics from this operator.
+	operatorNamespace, _ := k8sutil.GetOperatorNamespace()
```

Comment:
Previously, this had been set to the `WATCH_NAMESPACE` which was always `""`.  That resulted in:
```
time="2022-02-03T01:23:03Z" level=error msg="Could not create ServiceMonitor object. Error: an empty namespace may not be set during creation."
```

We could just keep that behavior by setting the `namespace` to `""`, but if we might as well comment it out.  If we want to enable this for the operator, we should turn it on for the namespace the operator is in, `k8sutil.GetOperatorNamespace()`